manifests/deployment_http.yaml

```diff
@@ -0,0 +1,80 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: wproofreader-app
+  labels:
+    helm.sh/chart: wproofreader-1.0.0
+    app.kubernetes.io/name: wproofreader
+    app.kubernetes.io/instance: wproofreader-app
+    app.kubernetes.io/version: "5.34.1"
```

Comment:
please update this to the latest app - 5.34.3

Reply:
Updated